### DISCUSSION TITLE
Can't get colormap_name to work

### DISCUSSION THREAD

**Discussion by daisy12321** [developmentseed/titiler#656] [Q&A] [answered]
This is a very basic question, but I can't seem to get `colormap_name` as a param to work. For example, I can get
`https://titiler.xyz/cog/tiles/WebMercatorQuad/14/8514/5834@1x?url=https%3A%2F%2Fdata.geo.admin.ch%2Fch.swisstopo.swissalti3d%2Fswissalti3d_2019_2573-1085%2Fswissalti3d_2019_2573-1085_0.5_2056_5728.tif`

to work, but not
`https://titiler.xyz/cog/tiles/WebMercatorQuad/14/8514/5834@1x?url=https%3A%2F%2Fdata.geo.admin.ch%2Fch.swisstopo.swissalti3d%2Fswissalti3d_2019_2573-1085%2Fswissalti3d_2019_2573-1085_0.5_2056_5728.tif&colormap_name=viridis`

am I missing something fundamental?

**Comment by vincentsarago** [comment, marked as answer]
@daisy12321 you need to rescale the data from any `min, max` to byte value (0->255) by passing `rescale=0,3000` 

see https://developmentseed.org/titiler/examples/notebooks/Working_with_CloudOptimizedGeoTIFF_simple/#display-tiles

**Comment by daisy12321** [comment]
Thank you for the helpful answer, it worked great.

Another follow up question @vincentsarago: is it possible to combine multiple bands from different files and show the RGB directly? For example, there are already files existing on S3 
"s3://sentinel-cogs/sentinel-s2-l2a-cogs/36/M/YE/2023/3/S2A_36MYE_20230302_0_L2A/B02.tif",
"s3://sentinel-cogs/sentinel-s2-l2a-cogs/36/M/YE/2023/3/S2A_36MYE_20230302_0_L2A/B03.tif",
"s3://sentinel-cogs/sentinel-s2-l2a-cogs/36/M/YE/2023/3/S2A_36MYE_20230302_0_L2A/B04.tif",

I can post-processing them and make a new 3-band RGB tif file on my S3 for TiTiler to render, but I am wondering if there are any clever ways to show the RGB directly based on the raw files?

**Reply by vincentsarago** [reply]
you can use STAC: https://developmentseed.org/titiler/examples/notebooks/Working_with_STAC_simple/

or create a custom a tiler using `S2COGReader` from https://cogeotiff.github.io/rio-tiler-pds/usage/sentinel/#cog-only-l2a-available-for-now and a `MultiBandTilerFactory` as shown in https://github.com/developmentseed/titiler-pds/blob/master/titiler_pds/routes/sentinel.py#L16-L21

**Reply by daisy12321** [reply]
That's super helpful. I will try those. Thanks!
